content/en/docs/components/multi-tenancy/getting-started.md

```diff
@@ -113,14 +113,14 @@ Kubeflow {{% kf-latest-version %}} provides automatic profile creation:
   - The Kubeflow deployment process automatically creates a profile for the user
     performing the deployment. When the user access the Kubeflow central dashboard
     they see their profile in the dropdown list.
-  - The automatic profile creation can be disabled as part of the deployment by setting the registration-flow env variable to false. And an admin can manually create profiles per user or per project and add collaborators through YAML files.
+  - The automatic profile creation can be enabled as part of the deployment by setting the registration-flow env variable to true. This variable is set to false by default. With registration-flow env variable set to false, an admin can manually create profiles per user or per project and add collaborators through YAML files.
```

Comment:
I'd remove the sentence `With registration-flow env variable set to false,` since it implies that an admin can manually create profiles only when this is set to false, which is not true.

Reply:
Then you can remove that line here and just say:
- Activate automatic user profile creation by setting the `registration-flow` environment variable to `true` during deployment.